deployment/cfn/public_hosted_zone.py

```diff
@@ -0,0 +1,47 @@
+from boto import route53 as r53
+
+from majorkirby import CustomActionNode
+
+
+class PublicHostedZone(CustomActionNode):
+    """Represents a Route53 private hosted zone"""
```

Comment:
Fixed in https://github.com/WikiWatershed/model-my-watershed/commit/423c8e3fe4f29e32c5a821582f5270d14ff69390.